### DISCUSSION TITLE
Can not register user

### DISCUSSION THREAD

**Discussion by QYQSDTC** [danny-avila/LibreChat#4405] [Troubleshooting]
### What happened?

When registering it shows there's something went wrong and the log shows
```
 error: [registerUser] Error in registering user: connect ECONNREFUSED 64.233.188.109:465
```

the same problem happens when resetting the password

### Steps to Reproduce

1. go to sign up a new account

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
error: [registerUser] Error in registering user: connect ECONNREFUSED 64.233.188.109:465
```

### Screenshots

![CleanShot-Xo1bgQF4@2x](https://github.com/user-attachments/assets/b99feb74-b6fd-4925-8c5e-fb55a8282ee8)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Not a bug and not enough information to help troubleshoot. Can you take a second look at .env.example and configure your .env file as needed?
